mypy/test/data.py

```diff
@@ -57,6 +58,8 @@ def parse_test_cases(
                         fnam = '__builtin__.py'
                     files.append((os.path.join(base_path, fnam), f.read()))
                     f.close()
+                elif p[i].id in ('expectStale', 'expectstale'):
```

Comment:
How about picking one case and sticking to it? Maybe even shorten to 'stale' (after all, we use `[out]`, not `[expectedOutput]` :-)